pyalex/api.py

```diff
@@ -3,6 +3,7 @@
 from urllib.parse import quote_plus
 
 import requests
+from urllib3.util import Retry
```

Comment:
Let's add to the dependencies even though it is a dep of requests. Just to be sure.

Reply:
Done